upf/upf.c

```diff
@@ -48,6 +48,13 @@
 
 #include <vppinfra/tw_timer_1t_3w_1024sl_ov.h>
 
+#if CLIB_DEBUG > 1
+#define upf_debug clib_warning
+#else
+#define upf_debug(...)                          \
+  do { } while (0)
+#endif
```

Comment:
`upf_debug` is not used in this file

Reply:
Sorry for pointing on debug code I've left, forgot to flush it
done